src/visual.ts

```diff
@@ -238,8 +281,67 @@ module powerbi.extensibility.visual {
             this.updateInternal(options);
         }
 
+        private getYAxisWidth(chartData: TableHeatMapChartData): number {
+            let maxLengthText: string = _.maxBy(chartData.categoryY, "length") || "";
+            maxLengthText = TableHeatMap.textLimit(maxLengthText, this.settings.yAxisLabels.maxTextSymbol);
+            return TextMeasurementService.measureSvgTextWidth({
+                fontSize: PixelConverter.toString(this.settings.yAxisLabels.fontSize),
+                text: maxLengthText.trim(),
+                fontFamily: this.mainGraphics.style("font-family")
```

Comment:
I'm not sure that take font family by using DOM each time is a good idea. Can we use an extra variable to get it?
This variable should keep font family.